undefined reference to `Free_Cseq'

Hi

I try to compile phyml 3.2.20160530 on Linux x86_64, gcc 4.9.1 (openMPI 1.8.1) but it fails:

```
sh ./autogen.sh
/configure --prefix=<...> --enable-phyml --enable-phytime
make
```

/bin/sh ../libtool  --tag=CC   --mode=link gcc  -Wall -O2 -msse -fomit-frame-pointer -funroll-loops -Wempty-body -Wuninitialized    -o phytime main.o utilities.o optimiz.o lk.o bionj.o models.o free.o help.o simu.o eigen.o pars.o alrt.o interface.o cl.o spr.o times.o m4.o draw.o rates.o mcmc.o stats.o mg.o tiporder.o io.o make.o mixt.o init.o nexus.o date.o xml.o -lm -lm -ldl
libtool: link: gcc -Wall -O2 -msse -fomit-frame-pointer -funroll-loops -Wempty-body -Wuninitialized -o phytime main.o utilities.o optimiz.o lk.o bionj.o models.o free.o help.o simu.o eigen.o pars.o alrt.o interface.o cl.o spr.o times.o m4.o draw.o rates.o mcmc.o stats.o mg.o tiporder.o io.o make.o mixt.o init.o nexus.o date.o xml.o  -lm -ldl
times.o: In function `TIMES_main':
times.c:(.text+0x4032): undefined reference to`Free_Cseq'
collect2: error: ld returned 1 exit status

with MPI enabled, same kind of issue:
/bin/sh ../libtool  --tag=CC   --mode=link mpicc  -Wall -O2 -msse -fomit-frame-pointer -funroll-loops -Wempty-body -Wuninitialized    -o phytime main.o utilities.o optimiz.o lk.o bionj.o models.o free.o help.o simu.o eigen.o pars.o alrt.o interface.o cl.o spr.o times.o m4.o draw.o rates.o mcmc.o stats.o mg.o tiporder.o io.o make.o mixt.o init.o nexus.o date.o xml.o -lm -lm 
libtool: link: mpicc -Wall -O2 -msse -fomit-frame-pointer -funroll-loops -Wempty-body -Wuninitialized -o phytime main.o utilities.o optimiz.o lk.o bionj.o models.o free.o help.o simu.o eigen.o pars.o alrt.o interface.o cl.o spr.o times.o m4.o draw.o rates.o mcmc.o stats.o mg.o tiporder.o io.o make.o mixt.o init.o nexus.o date.o xml.o  -lm
utilities.o: In function `Bootstrap_From_String':
utilities.c:(.text+0x2c4d1): undefined reference to`Bootstrap_MPI'
times.o: In function `TIMES_main':
times.c:(.text+0x407a): undefined reference to`Free_Cseq'
collect2: error: ld returned 1 exit status

How to solve this?
Regards

--- a/src/invitee.c
+++ b/src/invitee.c
@@ -688,7 +688,7 @@ void PhyTime_XML(char *xml_file)
   Free_Tree_Pars(tree);
   Free_Tree_Lk(tree);
   Free_Tree(tree);
-  Free_Cseq(cdata);
+  Free_Calign(cdata);
   Free_Model(mod);
   if(io -> fp_in_align)   fclose(io -> fp_in_align);
   if(io -> fp_in_tree)    fclose(io -> fp_in_tree);

